js/src/forum/components/PostStream.js

```diff
@@ -368,6 +374,10 @@ export default class PostStream extends Component {
    * @param {jQuery} $item
    */
   flashItem($item) {
-    $item.addClass('flash').one('animationend webkitAnimationEnd', () => $item.removeClass('flash'));
+    $item.addClass('flash').on('animationend webkitAnimationEnd', () => {
+      if (event.animationName === 'fadeIn') {
+        $item.removeClass('flash');
+      }
+    });
```

Comment:
Same here. This change was added in commit 1e95bb2a, but not extracted to master in #2262 and not reverted in commit 4872ddc5.

Is this needed or not? If so, what does it do? Should this be in master?

Reply:
SInce we now use CSS for everything, it ensures that we don't remove irrelevant animations (as animationend will run for any animation). And yes, it should be in master.

Reply:
I removed it here. Please feel to send a bugfix PR if this is still a problem in master.